### ISSUE TITLE
Create validation for related models

### ISSUE THREAD

**Issue by begin29** [DockYard/ember-validations#331]
I have model `user` and that belongs to `klass`  model. 

When I use simple validation for klass model as field it doesn't work. 
```js
klass: {
      presence: true
}
```

Also I tried to use `inline validation` like this: 
```js
klass: {
    inline: EmberValidations.validator(function() {
      var selectedKlass = this.get('selectedKlass');
      if ( Ember.isEmpty(selectedKlass) ) {
          return "can`t be blank";
      }
    })
}
```
But I always get validation error. Also with filled value. Ember.isEmpty(selectedKlass) returned false but message is returned. I don`t know how to deal with that. 

https://drive.google.com/file/d/0B-EmcLqIESFJbmpMemZGX0EydXc/edit?usp=drivesdk

**Comment by abobwhite** [comment]
@begin29 I know that it has been awhile since you opened this issue, but I just ran across the same issue and figured it out. Within a validator, you can use ```this.model``` to get access to the thing it's validating on (controller, model, component, etc.). So the name "model" is kind of generic in this case. If you were using the validator on a controller and needed to validate something on the model of that controller, you would use ```this.model.get('model')```.

Basically, in your validator, "this" is not the "this" you're looking for. *waves jedi hand*

Your code would look like this if "selectedKlass" is on the thing you're validating against:
```
klass: {
    inline: EmberValidations.validator(function() {
      var selectedKlass = this.model.get('selectedKlass');
      if ( Ember.isEmpty(selectedKlass) ) {
          return "can`t be blank";
      }
    })
}
```
